### PR TITLE
fix: 1183

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @MDunitz 

**Readability:** 

---

## Changes
- modify publish to ignore datasets with an original_uuid

### CODE REVIEW

File: tests/unit/backend/chalice/api_server/test_revisions.py [hunk 1]
```diff
@@ -232,57 +231,86 @@ def test__delete_published_dataset_during_revision(self):
...
+            ],
+            "data_submission_policy_version": "0",
+        }
+        self.rev_collection.update(**expected_body)
+        pub_s3_objects, rev_s3_objects = self.get_s3_objects_from_collections()
+        actual_body = self.publish_collection(self.rev_collection.id)
+        self.assertPublishedCollectionOK(expected_body, pub_s3_objects)
+        self.verify_datasets(actual_body, [ds.id for ds in self.pub_collection.datasets])
+
+    def test_with_revision_and_existing_datasets(self):
```
Comment (MDunitz) [inline tests/unit/backend/chalice/api_server/test_revisions.py:314]: Do you want to explicitly test that a newly created or refreshed dataset isnt published?

Comment (Bento007) [reply tests/unit/backend/chalice/api_server/test_revisions.py:314]: for that we need #1177, but I will add a test for that in that PR.

Comment (Bento007) [reply tests/unit/backend/chalice/api_server/test_revisions.py:314]: this won't actually bock. I removed that part. I'll add it in a separate PR.

**Review by MDunitz — APPROVED** [review]
lgtm

**Pull request merged**